### DISCUSSION TITLE
please help me with kade engine

### DISCUSSION THREAD

**Discussion by glowsoony** [KadeArchive/Kade-Engine#1610] [Q&A]
help i keep getting the same thing as Error: Source path "C:/HaxeToolkit/haxe/lib/extension-webm/git/ndll/Windows64/extension-webm.ndll" does not exist and i get this other one 
![image](https://user-images.githubusercontent.com/84847356/128451421-2b7563ec-05ff-4bb4-8ff6-3ad1aa17056d.png)

**Comment by napstaa967** [comment]
> help i keep getting the same thing as Error: Source path "C:/HaxeToolkit/haxe/lib/extension-webm/git/ndll/Windows64/extension-webm.ndll" does not exist and i get this other one
> ![image](https://user-images.githubusercontent.com/84847356/128451421-2b7563ec-05ff-4bb4-8ff6-3ad1aa17056d.png)

did you follow the build guide properly

as the guide says:
Install and set up the necessary libraries:
haxelib install lime 7.9.0
.
.
.
haxelib git extension-webm https://github.com/KadeDev/extension-webm
lime rebuild extension-webm windows

**Reply by glowsoony** [reply]
bro i tried that but something happend to my cmd because now im getting this
![image](https://user-images.githubusercontent.com/84847356/128452422-79b4a12a-f82e-4cd2-a310-23c719b36225.png)

**Reply by napstaa967** [reply]
> bro i tried that but something happend to my cmd because now im getting this
> ![image](https://user-images.githubusercontent.com/84847356/128452422-79b4a12a-f82e-4cd2-a310-23c719b36225.png)

try reinstalling lime

**Reply by glowsoony** [reply]
i did but it still the same :(

**Reply by SuperSpinDashGamer39** [reply]
uhhhhh try deleting for export folder

**Reply by glowsoony** [reply]
ok

**Reply by glowsoony** [reply]
export folder gone lol

**Reply by glowsoony** [reply]
welp now a i fixed it but still the same thing :/
![image](https://user-images.githubusercontent.com/84847356/128648106-aea9f184-d991-40a2-bffd-5fa525910968.png)
